org.jacoco.core.test/src/org/jacoco/core/runtime/ModifiedSystemClassRuntimeTest.java

```diff
@@ -82,8 +83,9 @@ public void should_instrument_java10_class() {
 	 * "java.lang.reflect.Module" introduced in JDK 9.
 	 */
 	private Instrumentation newInstrumentationMock() {
-		return (Instrumentation) Proxy.newProxyInstance(getClass()
-				.getClassLoader(), new Class[] { Instrumentation.class },
+		return (Instrumentation) Proxy.newProxyInstance(
```

Comment:
Minor remark: it took me some time during review to divide this and similar non-functional style changes from real ones. Wondering if we can improve this somehow later. As a note - usually I try to not reformat otherwise non-changed existing lines.